packages/@react-aria/selection/src/useSelectableList.ts

```diff
@@ -167,3 +167,21 @@ function scrollIntoView(scrollView: HTMLElement, element: HTMLElement) {
   scrollView.scrollLeft = x;
   scrollView.scrollTop = y;
 }
+
+/**
+ * Computes the offset left or top from child to ancestor by accumulating
+ * offsetLeft or offsetTop through intervening offsetParents.
+ */
+function relativeOffset(ancestor: HTMLElement, child: HTMLElement, axis: 'left'|'top') {
+  const prop = axis === 'left' ? 'offsetLeft' : 'offsetTop';
+  let sum = 0;
+  while (child.offsetParent) {
+    sum += child[prop];
+    if (child.offsetParent === ancestor) {
```

Comment:
I noticed that `child.offsetParent` in the menu resolved to the Popover div (kinda weird not sure why). Should this have another conditional that breaks out of the while if the offsetParent contains the ancestor?
```
if (child.offsetParent === ancestor || child.offsetParent.contains(ancestor))
```

Reply:
Oh good call!